docs/modules/ROOT/pages/references/architecture/control-api-user.adoc

```diff
@@ -0,0 +1,88 @@
+= {controlapi}: User
+
+TIP: This resource implements the xref:references/functional-requirements/portal.adoc#_feature_personal_settings_per_user["Personal Settings per User"] feature.
+
+== Object
+
+.CRD based
+[source,yaml]
+----
+apiVersion: appuio.io/v1
+kind: User
+metadata:
+  name: bec0d928-2ae2-4cec-94a0-5f72f12b8b39 <1>
+spec:
+  displayName: Kate Demo
+  username: kate.demo
```

Comment:
We decided to disalow changes to the `username` (https://github.com/appuio/appuio-io-docs/blob/8e54da6bf5cb4bf7f2154793115c50a1aaf12e7e/docs/modules/ROOT/pages/explanation/decisions/stable-usernames.adoc). Should this then be shown in `status` or would an admission controller take care of it?

Reply:
True that, updated accordingly.